src/test/operators/insert_test.cpp

```diff
@@ -228,9 +228,7 @@ TEST_F(OperatorsInsertTest, InsertNullIntoNonNull) {
 
   auto ins = std::make_shared<Insert>(t_name, gt2);
   auto context = TransactionManager::get().new_transaction_context();
-  ins->set_transaction_context(context);
```

Comment:
Aren't you testing different things now?

Reply:
The Insert will fail no matter what if the tx context is not set

Reply:
ah, right. Another reason why EXPECT_THROW without means to provide a means to actually test the value being thrown is a bad idea. Related: https://github.com/google/googletest/issues/952